packages/manager/src/components/AreaChart/AreaChart.tsx

```diff
@@ -41,7 +42,9 @@ interface AreaChartProps {
   areas: AreaProps[];
   ariaLabel: string;
   data: any;
+  fillOpacity?: number;
   height: number;
+  hideFill?: boolean;
```

Comment:
`hideFill` seems fine to me.  We could do something like this to potentially be more extendable / descriptive but it would accomplish the same thing. 

```suggestion
  /**
   * Makes the chart appear as a line or area chart
   * @default area
   */
  variant?: 'area' | 'line';
```

Reply:
@nikhagra-akamai Yea let's do this, I have a feeling we'll want this.